census_view.cpp

```diff
@@ -1468,6 +2588,12 @@ void CensusDVCView::UponUpdateNonemptySelection(wxUpdateUIEvent& e)
     e.Enable(0 < list_window_->GetSelections(selection));
 }
 
+void CensusGridView::UponUpdateNonemptySelection(wxUpdateUIEvent& e)
+{
+    auto const& selected_rows = grid_window_->GetSelectedRows();
+    e.Enable(!selected_rows.empty());
```

Comment:
_If_ we keep this function, it should just do `e.Enable(true)`, as "delete cells" menu item should be always enabled now.

Reply:
Looks like `UponUpdateNonemptySelection()` should be just deleted.

But if we trying to delete all cells, we recieve:
```
[Window Title]
Warning

[Content]
Cannot delete all cells. A census must always contain at least one cell.
[f:\prog\tt-solutions\lmi\census_view.cpp : 2871]

[OK]
```

May be this condition should be used (all selected or the table contains only one row) to update the delete button.

Reply:
I don't think it's a good idea as it wouldn't be clear why deleting the cells is not allowed, it's better to give an error message in this case. And, again, I prefer to not make any changes to the existing code unless they're absolutely needed.